docs/get-started.md

```diff
@@ -175,7 +175,7 @@ There is a bug where the Unity Editor does not import the navmesh for the `FPS-S
 To do this:
 
 1. Open the `FPS-SimulatedPlayerCoordinator` scene located at `Assets/Fps/Scenes`.
-2. Click on the `FPS-Start_Large` object in the hierarchy, and enable the object.
+2. Click on the `FPS-Start_Large` object in the Unity Scene Hierarchy window, and enable the object.
```

Comment:
maybe we link to the Unity manual here so those new to Unity can see the picture to say what exactly the `hierarchy` window is

https://docs.unity3d.com/Manual/Hierarchy.html